api/managementpb/log_level.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "api/managementpb;managementpb";
+
+// Log level for exporters
+enum LogLevel {
+  warn = 0;
+  debug = 1;
+  info = 2;
+  error = 3;
+  fatal = 4;
+}
```

Comment:
We breaking backward compatibility with such changes. Now you **should** specify log level. I think we need some `default = 0` value here. 

@BupycHuk what do you think?

Reply:
Agree, for better consistency we can call it auto like for metrics mode

Reply:
In issue https://jira.percona.com/browse/PMM-7326 :
```
while the default should be `–log–level=WARN`
```
so default = 0 = warn

Reply:
> Agree, for better consistency we can call it auto like for metrics mode

Like metrics mode:
```proto
enum LogLevel {
  default = 0;
  error = 1;
  fatal = 2;
  warn = 3;
  info = 4;
  debug = 5;
}
```

```go
// PUSH or AUTO variant enables pushMode for the agent.
func isPushMode(variant managementpb.MetricsMode) bool {
	return variant == managementpb.MetricsMode_PUSH || variant == managementpb.MetricsMode_AUTO
}
```

Proposal solution:
```
func specifyLogLevel(variant inventorypb.LogLevel) string {
	if variant == inventorypb.LogLevel_default {
		return ""
	}
	
	return variant.String()
}
```
```go
		row, err := models.CreateAgent(tx.Querier, models.MongoDBExporterType, &models.CreateAgentParams{
			PMMAgentID:        req.PmmAgentId,
			ServiceID:         service.ServiceID,
			Username:          req.Username,
			Password:          req.Password,
			AgentPassword:     req.AgentPassword,
			TLS:               req.Tls,
			TLSSkipVerify:     req.TlsSkipVerify,
			MongoDBOptions:    mongoDBOptions,
			PushMetrics:       isPushMode(req.MetricsMode),
			DisableCollectors: req.DisableCollectors,
			LogLevel:          specifyLogLevel(req.LogLevel),
		})
```

Reply:
yeah, with options in comments.

Reply:
> yeah, with options in comments.

done

Reply:
let's remove `log_level.*` from managementpb